blogposts/2021/java-scala-kotlin-code-intelligence.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Precise code intelligence for Java, Scala, and Kotlin'
+description: 'We are excited to announce that Sourcegraph now supports precise code intelligence for Java, Scala, and Kotlin.'
```

Comment:
```suggestion
description: 'We are excited to announce that Sourcegraph now supports precise code intelligence for Java, Scala, and Kotlin, enabling compiler-accurate “Go to definition” and “Find references” within a Git repository and all transitive dependencies of your codebase.'
```

Reply:
@olafurpg I didn't apply this change as I want to check the language makes sense. I wanted to add some more detail to the description so it's not as repetitive of the title

Reply:
It's great to have a longer description. The blog post was looking pretty empty from the `/blog/` overview page 
![image](https://user-images.githubusercontent.com/1408093/146784442-88862fa5-5d35-4adc-b488-8bdcfcb13418.png)